CHANGELOG.md

```diff
@@ -10,10 +10,21 @@ END_METADATA -->
 
 # Changelog
 
+![Vipps](./images/vipps.png) *Partner API is deprecated and will be phased out in Q4 2023.
```

Comment:
Nice. Requires contacting each user individually, since we are doing it quicker than in the API Lifecycle.